### PR TITLE
Fix main: ignore spell check Uncompress

### PR DESCRIPTION
**Pull request by gopidesupavan**
Spell check fix.

https://github.com/apache/airflow/actions/runs/11604744139/job/32314423307#step:7:10728

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Comment by gopidesupavan** [comment]
Came from here https://github.com/apache/airflow/pull/43526

**Review by amoghrajesh — APPROVED** [review]
Isn't the spelling correct :) 
LGTM anyways

File: docs/spelling_wordlist.txt [hunk 1]
```diff
@@ -1734,6 +1734,7 @@ unarchived
 unassigns
 uncomment
 uncommenting
+Uncompress
```
Comment (eladkal) [inline docs/spelling_wordlist.txt]: Is that a right word? should we just replace uncompress with decompress?
Maybe native english speaker should review this

Comment (gopidesupavan) [reply docs/spelling_wordlist.txt]: Both are synonyms, i am okay with either :)  yes please suggest..

Comment (uranusjr) [reply docs/spelling_wordlist.txt]: I believe decompress is more widely used, but some services like Snowflake does use *uncompress*, so we don’t really have much choice but to add this. There’s actually one usage we can control in `airflow.utils.compression`, but it’s probably not worthwhile.

By the way, this should likely be lowercase. I think our spell checking tools now is case insensitive anyway.

**Review by eladkal — APPROVED** [review]

File: docs/spelling_wordlist.txt (hunk 1 above)
Comment (gopidesupavan) [reply docs/spelling_wordlist.txt]: done :)

**Review by amoghrajesh — APPROVED** [review]

**Review by Lee-W — APPROVED** [review]

**Review by potiuk — APPROVED** [review]

**Pull request merged**